src/pages/en/guides/migrate-to-astro/from-nuxtjs.mdx

```diff
@@ -2,68 +2,682 @@
+Then, copy your existing Nuxt project files over to your new Astro project in a separate folder outside of `src`.
 
-While file-based routing and layout components are similar in Astro, you may wish to read about [Astro's project structure](/en/core-concepts/project-structure/) to learn where files should be located. For example, Astro uses a `public/` folder outside of `src/` for static assets.
+:::tip
+Prefer to explore in your browser instead? Visit https://astro.new for a variety of starter templates that can be opened and edited in StackBlitz, CodeSandbox or Gitpod.
+:::
 
-To convert other types of sites, such as a portfolio or documentation site, see more official starter templates on [astro.new](https://astro.new). You'll find a link to each project's GitHub repository, as well as one-click links to open a working project in StackBlitz, CodeSandbox and Gitpod online development environments.
+### Install integrations (optional)
 
-## Community Resources
+You may find it useful to install some of [Astro's optional integrations](/en/guides/integrations-guide/) to use while converting your Nuxt project to Astro:
 
-- Blog Post: [From Nuxt to Astro - rebuilding with Astro](https://dev.to/lindsaykwardell/from-nuxt-to-astro-rebuilding-with-astro-5ann)
+- **@astrojs/vue**: to reuse some existing Vue UI components in your new Astro site, or keep writing with Vue components.
+
+- **@astrojs/image**: to replace Nuxt's Image plugin with Astro's own image-optimizing components.
+
+- **@astrojs/mdx**: to bring existing MDX files from your Nuxt project, or to use MDX in your new Astro site.
+
+### Put your source code in `src`
+
+1. **Move** Nuxt's `static/` folder to `public/`. 
+   
+    Astro uses the `public/` directory for static assets, similar to Nuxt's `static/` folder.
+
+2. **Copy or Move** Nuxt's other files and folders (e.g. `pages`, `layouts` etc.) into Astro's `src/` folder.
+
+    Like Nuxt, Astro's `src/pages/` folder is a special folder used for file-based routing. All other folders are optional, and you can organize the contents of your `src/` folder any way you like. Other common folders in Astro projects include `src/layouts/`, `src/components`, `src/styles`, `src/scripts`.
+
+
+### Repurpose config file
+
+Astro has a configuration file at the root of your project called [`astro.config.mjs`](/en/guides/configuring-astro/). This is used only for configuring your Astro project and any installed integrations, including [SSR adapters](/en/guides/deploy/).
+
+### Convert Vue SFC files to `.astro` files
+
+Here are some tips for converting a Nuxt `.vue` component into a `.astro` component:
+
+1. Use the `<template>` of the existing NuxtJS component function as the basis for your HTML template.
+
+2. Change any [Nuxt or Vue syntax to Astro](#convert-nuxtjs-syntax-to-astro) or to HTML web standards. This includes `<NextLink>`, `:class`, `{{variable}}`, and `v-if`, for example.
+
+1. Move `<script>` JavaScript, into a "code fence" (`---`).
```

Comment:
The syntax in a Nuxt `<script>` section will be very Vue-like. Just like the Vue markup syntax had to be converted in step 2, the Vue `<script>` component syntax [(a big object with lots of options)](https://nuxtjs.org/docs/directory-structure/pages/#properties) needs to be converted into something Astro understands. This is covered well at the end of the guided example at the end, but may be worth a mention here.

To @sarah11918's point, sometimes code in the `<script>` tag should be converted into a `<script>` tag in Astro - for example, if there's an `onMount()` hook in the Vue SFC, that should be a `<script>` tag in `.astro`. But this might need its own section - it's hard to describe without a code sample.

```suggestion
1. Move `<script>` JavaScript, into a "code fence" (`---`). Convert your component's data-fetching properties to server-side JavaScript - see [Nuxt data fetching to Astro](#nuxt-data-fetching-to-astro). 
```

Reply:
I generally think this suggestion fits the bill well enough. @sarah11918 and I discussed fairly in-depth that there should only be one primary code example in each guide to cut down on size, and I don't know how to easily expand our code sample for Nuxt 2 to reflect this.